packages/security/src/DefaultCredentialManager.ts

```diff
@@ -100,24 +100,15 @@ export class DefaultCredentialManager extends AbstractCredentialManager {
     /* Gather all services. We will load secure properties for the first
      * successful service found in the order that they are placed in this array.
      */
-    this.allServices = [this.service];
+    this.allServices = [service || DefaultCredentialManager.SVC_NAME];
 
-    /* In our current implementation, this.service is always
-     * DefaultCredentialManager.SVC_NAME. We keep the logic below,
-     * in case we re-enable overrides in the future.
-     */
-    if (this.service !== DefaultCredentialManager.SVC_NAME) {
-        this.allServices.push(DefaultCredentialManager.SVC_NAME);
+    if (this.defaultService === DefaultCredentialManager.SVC_NAME) {
```

Comment:
so if the user specified they own service in which to store credentials, we are not going to load any of the other services?